packages/wrangler/src/__tests__/publish.test.ts

```diff
@@ -3184,6 +3184,67 @@ addEventListener('fetch', event => {});`
     });
   });
 
+  describe("[define]", () => {
+    it("should be able to define values that will be substituted", async () => {
```

Comment:
How about:

```suggestion
    it("should be able to define values that will be substituted into top-level identifiers", async () => {
```

and then add an instance of an identifier that will not be substituted.

Reply:
done and done